src/__private_stories__/private-responsive-layout-scenarios-story.tsx

```diff
@@ -0,0 +1,230 @@
+import * as React from 'react';
+import {
+    ResponsiveLayout,
+    Slideshow,
+    Hero,
+    Image,
+    ButtonPrimary,
+    ButtonLink,
+    Stack,
+    Placeholder,
+    Box,
+    Carousel,
+    MediaCard,
+} from '..';
+import {InternalResponsiveLayout, ResetResponsiveLayout} from '../responsive-layout';
+import usingVrImg from '../__stories__/images/using-vr.jpg';
+
+export default {
+    title: 'Private/ResponsiveLayout scenarios',
+    parameters: {fullScreen: true},
+};
+
+export const Default: StoryComponent = () => {
+    return (
+        <Box paddingY={24}>
+            <Stack space={24}>
+                {/** Carousel with alternating fullWidth and not fullWidth layouts */}
+                <ResponsiveLayout>
+                    <ResponsiveLayout fullWidth>
+                        <ResponsiveLayout>
+                            <ResponsiveLayout fullWidth>
+                                <ResponsiveLayout>
+                                    <ResponsiveLayout fullWidth>
```

Comment:
what are we testing here? do we need to repeat the nesting so many times?
it's the carousel relevant to the test or we could use a simpler component (Placeholder)?

Reply:
The carousel is relevant because it uses the margin var for some calculations. I could remove some of the nested layouts, but I also don't see any harm in leaving them as they are